pkg/maps/lbmap/types.go

```diff
@@ -223,7 +223,7 @@ type Backend interface {
 	Map() *bpf.Map
 
 	// Get backend identifier
-	GetID() loadbalancer.BackendID
+	GetID() uint32
```

Comment:
Why are we changing this from `loadbalancer.BackendID` to `uint32` if the type `loadbalancer.BackendID` is set to `uint32`? (Same question applies to most of the changes in this commit)

Reply:
The bug which this PR tries to fix was introduced because I decided to reuse the type definition. IMO, to prevent from such mishaps in the future, it's better to avoid representing BPF maps in Go with "higher-level" types.